src/controllers/log.controller.ts

```diff
@@ -0,0 +1,141 @@
+import { logDB } from '../models/db.index';
+import { RequestHandler } from 'express';
+import createHttpError from 'http-errors';
+
+export const getLogs: RequestHandler = async (req, res, errFunc) => {
+    try {
+        const logs = await logDB.findAll();
+        res.status(200).json(logs);
+    } catch (error) {
+        errFunc(error);
+    }
+};
+
+export const getLog: RequestHandler = async (req, res, errFunc) => {
```

Comment:
same as previous comment, honestly the sending reference data probably only applies here, not the multi get